frontend/src/app/app.component.html

```diff
@@ -1 +1,5 @@
 <router-outlet></router-outlet>
+<label class="switch">
```

Comment:
Can we move this functionality into the header?

Reply:
done.